CHANGELOG.rst

```diff
@@ -4,10 +4,11 @@ Changelog
 This document describes changes between each past release.
 
 
-1.7.0 (unreleased)
+1.7.0 (2015-10-28)
 ==================
 
-- Nothing changed yet.
+- Upgraded to *Cliquet* 2.9.0
```

Comment:
Add a link *more details* to here https://github.com/mozilla-services/cliquet/releases/tag/2.9.0 ?